examples/basics/plotting/grid_x_y_viewbox.py

```diff
@@ -0,0 +1,47 @@
+
+
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
++---+-------------+
+|   |             |
+| y |   viewbox   |
+|   |             |
++---+-------------+
+| sp|     x       |
++---+-------------+
+"""
+
+import sys
+
+from vispy import scene, app
+
+canvas = scene.SceneCanvas(keys='interactive')
+canvas.size = 600, 600
+canvas.show()
+
+grid = canvas.central_widget.add_grid()
+
+widget_y_axis = grid.add_widget(row=0, col=0)
+widget_y_axis.bgcolor = "#999999"
+widget_y_axis.stretch = (0.2, 1)
```

Comment:
I just tried setting:

    widget_y_axis.width_min = 50
    widget_y_axis.width_max = 50
    widget_x_axis.height_min = 50
    widget_x_axis.height_max = 50

..and it seems to work well :)  Can you write that into the example in place of `x_axis.stretch` and `y_axis.stretch`?